app/services/submission_service.rb

```diff
@@ -30,6 +31,12 @@ def create_submission(
       user.session_id = submission_params.delete(:session_id)
       create_params = submission_params.merge(user_id: user.id)
 
+      if AdminUser.exists?(gravity_user_id: current_user)
+        create_params.merge!(
+          created_by: User.find_by(gravity_user_id: current_user).email
+        )
+      end
```

Comment:
```suggestion
      create_params = submission_params.merge(
        user_id: user.id,
        created_by: User.find_by(gravity_user_id: current_user)&.email
      )
```

Reply:
oh, do we save the user's email only if he is one of the admins?

Reply:
if yes - undo my changes :)

Reply:
oh hey yes, we want to save the admin's email only to the created_by field, and leave it nil if the operator is not an admin